### PR TITLE
Fxos icons

### PR DESCRIPTION
**Pull request by jonathanKingston**
Extends from: https://github.com/etiennesegonzac/hope/pull/11

The URL bar icons I kept the same as the reload icon looks quite different and the reader icon doesn't feature in the fxos-icons set.

I also think the home button would look better as something else also.

### CODE REVIEW

**Comment by etiennesegonzac** [comment]
Hey @jonathanKingston can you rebase and ask @wilsonpage for review ? :)

**Comment by jonathanKingston** [comment]
Sure, I noticed an alignment issue also so will fix.. not sure why it working before.

**Comment by jonathanKingston** [comment]
Hey @wilsonpage when you get time would you be able to review this?
Perhaps after you have recovered from new year :smile:

I changed the url bar icon as it doesn't feature in the font. Also not sure of the home icon (it appeared to be double width so I had to force it's position) I also like the idea of the firefox icon there.

File: addon.js [hunk 1]
```diff
@@ -8,7 +8,14 @@
     '#statusbar.light { filter:none !important; }',
     '#windows .appWindow .titlebar { background: #333333 !important }',
     '#windows .appWindow.homescreen { display:none !important; animation: none !important }',
-    '#software-home-ring { outline: none; background: no-repeat center/70% url(' + browser.extension.getURL('assets/Home.png') + '); }',
+    '#software-home-ring {',
+    'outline: none;',
+    '--ring-height-width: inherit;',
```
Comment (wilsonpage) [inline addon.js:13]: What is this var used for?

File: lib/hope-tab.js [hunk 2]
```diff
@@ -123,9 +123,9 @@ window.HopeTab = fxosComponent.register('hope-tab', {
         </div>
         <div class="url">
           <div class="field">
-            <button class="reader"></button>
+            <button class="reader" data-icon="more" aria-label="more" ></button>
             <div class="url-text"></div>
-            <button class="refresh"></button>
+            <button class="refresh" data-icon="reload" aria-label="reload" ></button>
```
Comment (wilsonpage) [inline lib/hope-tab.js:128]: Where is `fxos-icons.css` loaded into the document?

File: lib/hope-tab.js [hunk 3]
```diff
@@ -243,14 +245,6 @@ window.HopeTab = fxosComponent.register('hope-tab', {
         box-sizing: content-box;
       }
 
-      .url .field .reader {
-        background-image: url(assets/urlbar_reader_mode_icon.png);
-      }
-
-      .url .field .refresh {
-        background-image: url(assets/urlbar_refresh_icon.png);
```
Comment (wilsonpage) [inline lib/hope-tab.js:251]: If UX want these specific icons I think we need to get them added to fxos-icons (or perhaps a 'next' fork) before we land this change :)

**Comment by wilsonpage** [comment]
Thanks for this @jonathanKingston! A few inline questions. I've also emailed one of the visual chaps to ask for the new SVG assets. Instead of bloating the current fxos-icons set, I think it would make sense to fork and replace icons as and when required.

File: addon.js (hunk 1 above)
Comment (jonathanKingston) [inline addon.js:13]: Something native on the existing button, I'll figure out what it is used for and use the property instead.

File: lib/hope-tab.js (hunk 2 above)
Comment (jonathanKingston) [inline lib/hope-tab.js:128]: Already was being loaded:
https://github.com/etiennesegonzac/hope/blob/995f01832fd85562dbbb14c3d0fecb07108c39d0/index.html#L7

I have just added it for all other icons etc.

File: lib/hope-tab.js (hunk 3 above)
Comment (jonathanKingston) [inline lib/hope-tab.js:251]: Good plan the reader one certainly looks much better. Not sure what to do for the home icon, it was just a circle before but neither the new one I used right.

**Comment by jonathanKingston** [comment]
@wilsonpage thanks for the review, I was just scouting about Mozilla repos to learn the ecosystem more. Feel free to tear this up etc if it's too much work to put in. The main issue I had as you mention is the missing icons which certainly would be worth adding to the icons either under new icon names or as a branch etc as you mention.

On holiday at the moment so might be light on the updates etc.

**Comment by jonathanKingston** [comment]
Closing in favour of: https://github.com/etiennesegonzac/hope/pull/22

**Pull request closed without merging**